<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>830</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>야놀자 야체 B</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>서울부터 제주까지,</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_name">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>224</y>
     <width>270</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>야놀자 야체 R</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(255, 0, 127);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_exec">
   <property name="geometry">
    <rect>
     <x>327</x>
     <y>230</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>야놀자 야체 R</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>검색</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmb_hotel_name">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>370</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>야놀자 야체 R</family>
     <pointsize>18</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_area">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>334</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>야놀자 야체 R</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>인기 지역</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>372</y>
     <width>370</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_4">
      <property name="font">
       <font>
        <family>야놀자 야체 R</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>부산</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_6">
      <property name="font">
       <font>
        <family>야놀자 야체 R</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>제주</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_5">
      <property name="font">
       <font>
        <family>야놀자 야체 R</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>강릉</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_7">
      <property name="font">
       <font>
        <family>야놀자 야체 R</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>서울</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="font">
       <font>
        <family>야놀자 야체 R</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>거제</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="font">
       <font>
        <family>야놀자 야체 R</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>인천</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="font">
       <font>
        <family>야놀자 야체 R</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>여수</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lb_recommend">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>420</y>
     <width>380</width>
     <height>290</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>야놀자 야체 R</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>추천 호텔</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>130</y>
     <width>60</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>C:/Users/013/Documents/카카오톡 받은 파일/KakaoTalk_20200929_143022419_07.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>162</x>
     <y>170</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>야놀자 야체 R</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>다양한 숙소가 한 번에!</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>431</width>
     <height>790</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>C:/Users/013/Documents/카카오톡 받은 파일/KakaoTalk_20200929_143022419_04.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>le_name</zorder>
  <zorder>btn_exec</zorder>
  <zorder>cmb_hotel_name</zorder>
  <zorder>btn_area</zorder>
  <zorder>horizontalLayoutWidget</zorder>
  <zorder>lb_recommend</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
